lib/tests/streamlit/legacy_add_rows_test.py

```diff
@@ -303,3 +306,53 @@ def test_legacy_add_rows_with_pyarrow_table_data(self):
             # Clear the queue so the next loop is like a brand new test.
             get_script_run_ctx().reset()
             self.forward_msg_queue.clear()
+
+
+def _get_data_frame(delta: Delta, name: Optional[str] = None) -> DataFrame:
+    """Extract the dataframe protobuf from a delta protobuf."""
```

Comment:
This function used to live in `legacy_data_frame.py`, where it was called by runtime code. It's now only used by tests, so I moved it here (and removed its associated tests).